dev/com.ibm.ws.ui.tool.serverConfig/resources/WEB-CONTENT/js/components/core.js

```diff
@@ -9,909 +9,753 @@
+             xhr.setRequestHeader("com.ibm.websphere.jmx.connector.rest.routing.serverName", serverName);
+             xhr.setRequestHeader("com.ibm.websphere.jmx.connector.rest.routing.serverUserDir", userDir);
+         }        
+     };
+ 
+ 
+     // Apply the three custom headers for REST API calls to roots that start with /ibm/api/
+     var applyRestRoutingHeaders = function(xhr) {
+         var hostName = collectiveServerData.host;
+         var serverUserDir = collectiveServerData.wlpUserDir;
+         var serverName = collectiveServerData.name;
+ 
+         xhr.setRequestHeader("collective.hostNames", hostName);
+         xhr.setRequestHeader("collective.serverNames", serverName);
+         xhr.setRequestHeader("collective.serverUserDirs", serverUserDir);
+     };
+     
+     
+     var centerDialog = function(modalElement) {
+         var modal = $(modalElement);
+         var dialog = modal.find(".modal-dialog");
+         modal.css("display", "block");
+         dialog.css("margin-top", 40 + Math.max(0, ($(window).height() - dialog.height()) / 4));
+     };
+     
+     
+     var setUIBlocked = function(blocked) {
+         uiBlocked = blocked;
+         if(uiBlocked) {
+             showControlById("uiBlock");
+         } else {
+             hideControlById("uiBlock");
+         }
+     };
+     
+     
+     var isInIFrame = function() {
+         return window.frameElement !== null && window.frameElement !== undefined;
+     };
+     
+     
+     var encodeServerId = function(serverId) {
+         var firstCommaIndex = serverId.indexOf(",");
+         var lastCommaIndex = serverId.lastIndexOf(",");
+ 
+         var hostName = serverId.substring(0, firstCommaIndex);
+         var userDir = serverId.substring(firstCommaIndex + 1, lastCommaIndex);
+         var serverName = serverId.substring(lastCommaIndex + 1, serverId.length);
+         
+         return hostName + "," + encodeURIComponent(userDir) + "," + serverName;
+     };
+ 
+ 
+     var collectiveRoutingRequired = function() {
+         return environment === constants.ENVIRONMENT_COLLECTIVE && serverId !== hostServerId;
+     };
+ 
+ 
+     $(document).ready(function() {
+         
+         $(window).on("popstate", function() {
+             var skip = false;
+             if(embedded !== null && embedded !== undefined && window.top.location.hash.length === 0) {
+                 skip = true;
+             }
+             if(!skip) {
+                 if(embedded === constants.EMBEDDED_EXPLORE_TOOL) {
+                     var filePath = getDataFromURLHash().filePath;
+                     if(environment === constants.ENVIRONMENT_COLLECTIVE) {
+ //                        window.top.history.pushState(null, null, "#" + constants.EXPLORE_TOOL_HASH + "/" + constants.EXPLORE_TOOL_SERVERS_SEGMENT + "/" + serverId + "/" + constants.EXPLORE_TOOL_CONFIGURE_SEGMENT + (filePath !== null && filePath !== undefined? "/" + filePath : ""));
```

Comment:
Do we need to keep commented out code here? Should this be removed?

Reply:
Resolving because this is not new code

Reply:
Commented line removed.